.github/workflows/ateam.yml

```diff
@@ -6,6 +6,33 @@ on:
       - main
 
 jobs:
+  aws_setup:
+    if: github.event.pull_request.head.repo.full_name == github.repository
+
+    name: setup AWS resources
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: change directory
+        run: cd tform/aws-resources
+
+      - name: Terraform setup
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terrarform_version: 0.14.9
```

Comment:
Does this step requires the change of directory executed before?

Reply:
This commit was a bit outdated, I was coming up on an error where terraform couldn't find the plan, I thought it was due to it being in the wrong directory but the error turned out to be something else, which I wanted to bring up on our next standup since it requires a bit of discussion